docs/release-notes/8.17.asciidoc

```diff
@@ -1,6 +1,59 @@
 [[release-notes-header-8.17.0]]
 == 8.17
 
+[discrete]
+[[release-notes-8.17.2]]
+=== 8.17.2
+
+[discrete]
+[[known-issue-8.17.2]]
+==== Known issues
+
+// tag::known-issue[]
+[discrete]
+.Duplicate alerts can be produced from manually running threshold rules 
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running threshold rules could produce duplicate alerts if the date range was already covered by a scheduled rule execution.
+====
+// end::known-issue[]
+
+// tag::known-issue[]
+[discrete]
+.Manually running custom query rules with suppression could suppress more alerts than expected
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running a custom query rule with suppression could incorrectly inflate the number of suppressed alerts. 
+====
+// end::known-issue[]
+
+[discrete]
+[[features-8.17.2]]
+==== New features
+* Adds the `advanced.malware.max_file_size_bytes` <<adv-policy-settings,advanced policy setting>>, which allows you to control the maximum file size for malware protection.
+
+[discrete]
+[[enhancements-8.17.2]]
+==== Enhancements
+* Enhances the performance of {elastic-defend} network events monitoring for better CPU utilization and responsiveness.
+
+[discrete]
+[[bug-fixes-8.17.2]]
+==== Bug fixes
+* Ensures that multiple IPs are displayed as individual links in the Alerts table, even if they're passed as a single string ({kibana-pull}209475[#209475]).
+* Fixes an Attack discovery bug that prevented you from selecting different connector types after initially choosing one ({kibana-pull}208969[#208969]).
```

Comment:
This is not an Attack discovery bug, but rather AI assistant within security solution. It also happens on other pages where you can select connectors.

Reply:
```suggestion
* Fixes an AI Assistant bug that prevented you from selecting different connector types after initially choosing one ({kibana-pull}208969[#208969]).
```